src/fragments/start/getting-started/next/setup.mdx

```diff
@@ -1,9 +1,13 @@
+<Callout>
+Amplify JavaScript works best with the Next.js <a href="https://nextjs.org/docs/pages/building-your-application">Pages Router</a>. Full support for the App Router introduced in Next.js 13.4 is coming soon.
```

Comment:
Note: I had to use an `<a>` tag here since it's inside of a React component. Tried using MDX style and it was rendering the brackets

Reply:
Nevermind, I fixed it by adding extra lines